### PR TITLE
add couple of FAQ items to cloud SSO FAQ

### PR DESCRIPTION
**Pull request by MichaelTrestman**
https://cockroachlabs.atlassian.net/browse/DOC-5308

### CODE REVIEW

File: cockroachcloud/cloud-sso.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ With support for SAML and OIDC, {{ site.data.products.db }} allows enterprise or
 
 ### Custom sign-in page
 
-With this feature, customers get a unique and private sign-in URL for their {{ site.data.products.db }} organization, and will no longer use the [public sign-in URL](https://cockroachlabs.cloud).
+With Enterprise Authentication, customers get a unique and private sign-in URL for their {{ site.data.products.db }} organization, and will no longer use the [public sign-in URL](https://cockroachlabs.cloud).
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:28]:
```suggestion
When {{ site.data.products.enterprise }} Authentication is enabled, members of your {{ site.data.products.db }} 
 organization use a unique and private sign-in page and can no longer sign in using the [public sign-in URL](https://cockroachlabs.cloud).
```
A little rewording to make this a bit more direct, reflect the same language as the section title, and avoid future tense (and fix the linter problem)

File: cockroachcloud/cloud-sso.md [hunk 2]
```diff
@@ -70,3 +70,11 @@ To remove a user's access to {{ site.data.products.db }} without deleting their
 ### Can admins require a particular login method for their {{ site.data.products.db }} organization?
 
 Yes, as long as Enterprise Authentication is enabled for your {{ site.data.products.db }} organization.
+
+### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
+
+Enterprise Authentication currently supports only a Service Provider (CockroachDB Cloud) initiated flow. The identity provider initiated flow can not be enabled yet.
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:76]:
```suggestion
{{ site.data.products.enterprise }} Authentication supports only a flow initiated by {{ site.data.products.db }}. Initiation from the IdP is not supported.
```
Avoiding future promises and a bit more direct, as long as this is accurate.

File: cockroachcloud/cloud-sso.md [hunk 3]
```diff
@@ -70,3 +70,11 @@ To remove a user's access to {{ site.data.products.db }} without deleting their
...
+
+### What default role is assigned to users when autoprovisioning is enabled in a CockroachDB Cloud org?
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:78]:
```suggestion
### What default role is assigned to users when auto-provisioning is enabled in a {{ site.data.products.db }} organization?
```
Is the feature. called "autoprovisioning" (one word? I've seen it with a hyphen so far. If it's really the former, we need to add it to the linter and make sure we are consistent. We should use the variables for Cloud and Enterprise consistently.

File: cockroachcloud/cloud-sso.md [hunk 4]
```diff
@@ -70,3 +70,11 @@ To remove a user's access to {{ site.data.products.db }} without deleting their
...
+
+`Developer role` is assigned by default to autoprovisioned users.
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:80]:
```suggestion
The `Developer` role is assigned by default to autoprovisioned users.
```
Same question for "autoprovisioning"

**Review by mdlinville — COMMENTED** [review]
Some small suggestions for consistency and clarity, but feel free to edit my edits if they are incorrect.

File: cockroachcloud/cloud-sso.md [hunk 5]
```diff
@@ -73,8 +73,8 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
 
 ### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:74]: Please address this linter warning too.

File: cockroachcloud/cloud-sso.md [hunk 6]
```diff
@@ -73,8 +73,8 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
...
 
-Enterprise Authentication currently supports only a Service Provider (CockroachDB Cloud) initiated flow. The identity provider initiated flow can not be enabled yet.
+{{ site.data.products.enterprise }} Authentication supports only a flow initiated by {{ site.data.products.db }}. Initiation from the IdP is not supported.
 
-### What default role is assigned to users when autoprovisioning is enabled in a CockroachDB Cloud org?
+### What default role is assigned to users when auto-provisioning is enabled in a {{ site.data.products.db }} organization?
 
-`Developer role` is assigned by default to autoprovisioned users.
+The `Developer` role is assigned by default to autoprovisioned users.
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:80]: Please either use "auto-provisioning" everywhere or add "autoprovisioning" to the dictionary file and use that everywhere.

**Review by mdlinville — CHANGES_REQUESTED** [review]
Let's address these remaining linter issues.

File: cockroachcloud/cloud-sso.md (hunk 5 above)
Comment (MichaelTrestman) [reply cockroachcloud/cloud-sso.md:74]: It does not apply here. The name of the feature is "Enterprise Authentication", that does not change if we change the name "Enterprise", which is orthogonal to Enterprise Authentication. Is there a way to one off dismiss or over-ride the linter warning?

File: cockroachcloud/cloud-sso.md [hunk 7]
```diff
@@ -70,3 +70,11 @@ To remove a user's access to {{ site.data.products.db }} without deleting their
...
 
 Yes, as long as Enterprise Authentication is enabled for your {{ site.data.products.db }} organization.
+
+### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
+
+{{ site.data.products.enterprise }} Authentication supports only a flow initiated by {{ site.data.products.db }}. Initiation from the IdP is not supported.
+
+### What default role is assigned to users when auto-provisioning is enabled in a {{ site.data.products.db }} organization?
+
+The `Developer` role is assigned by default to autoprovisioned users.
```
Comment (MichaelTrestman) [inline cockroachcloud/cloud-sso.md:80]:
```suggestion
The `Developer` role is assigned by default to auto-provisioned users.
```

File: cockroachcloud/cloud-sso.md (hunk 6 above)
Comment (MichaelTrestman) [reply cockroachcloud/cloud-sso.md:80]: done.

File: cockroachcloud/cloud-sso.md (hunk 5 above)
Comment (mdlinville) [reply cockroachcloud/cloud-sso.md:74]: The variable resolves to the word Enterprise so I've been using it in these cases. Another option is to create a Liquid variable for the name of this feature and to use that everywhere. Maybe Nick has more ideas.

**Review by mdlinville — APPROVED** [review]

**Pull request merged**